src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleasePermissionService.cs

```diff
@@ -98,7 +124,8 @@ public class ReleasePermissionService : IReleasePermissionService
                         .Distinct()
                         .ToListAsync();
 
-                    return users
+                    return await users
+                        .ToAsyncEnumerable()
```

Comment:
Same as above - we're not doing any async work here so there's no need to convert this to an async enumerable, and we can terminate with `ToList` without needing to await.